### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
In submitting this contribution, I agree to the current Software AG contributor agreement as referred to here: 
https://github.com/quartz-scheduler/contributing/blob/main/CONTRIBUTING.md

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/quartz/actions/runs/5223305049#summary-14139916034

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/quartz/actions/runs/5223305713#summary-14139918238

### API Breaks
*  org.quartz.JobDetail.isConcurrentExectionDisallowed -> isConcurrentExecutionDisallowed
* org.quartz.core.QuartzScheduler.notifySchedulerListenersSchduled -> notifySchedulerListenersScheduled
* org.quartz.impl.JobDetailImpl.isConcurrentExectionDisallowed -> isConcurrentExecutionDisallowed
* org.quartz.impl.org.quartz.impl.PROP_DATASOURCE_JNDI_PROVDER -> PROP_DATASOURCE_JNDI_PROVIDER
* org.quartz.impl.jdbcjobstore.AttributeRestoringConnectionInvocationHandler. restoreOriginalAtributes -> restoreOriginalAttributes
* org.quartz.impl.jdbcjobstore.JTANonClusteredSemaphore. DEFAULT_TRANSACTION_MANANGER_LOCATION -> DEFAULT_TRANSACTION_MANAGER_LOCATION
* org.quartz.utils.CircularLossyQueue.isEmtpy -> isEmpty

## Checklist
- [ ] tested locally
- [ ] updated the docs
- [ ] added appropriate test -- I offer the workflow I use separately if people are interesteed
- [x] signed-off on the above mentioned SoftwareAG contributor agreement via `git commit -s` on my commits. 
  (If you're not using command-line, you can use a [browser extension](https://github.com/scottrigby/dco-gh-ui) )

Fixes #

### CODE REVIEW

File: docs/changelog.adoc [hunk 1]
```diff
@@ -106,7 +106,7 @@ quartz-1.7.3 	24-Feb-2010
 quartz-1.7.2 	10-Feb-2010 
 ----
 
-See https://jira.terracotta.org/jira/browse/QTZ/?selectedTab=com.atlassian.jira.jira-projects-plugin:changelog-panel[older JIRA for older release changelog].
+See https://jira.terracotta.org/jira/browse/QTZ/?selectedTab=com.atlassian.jira.jira-projects-plugin:changelog-panel[older Jira for older release changelog].
```
Comment (jsoref) [inline docs/changelog.adoc:109]: Jira rebranded a while ago, and Atlassian products will actively change `JIRA` to `Jira`

File: docs/cookbook/BiDailyTrigger.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ active_sub_menu_id: site_mnu_docs_cookbook
 
 # How-To: Trigger That Executes Every 2 Days
 
-At first glance, you may be tempted to use a CronTrigger. However, if this is truly to be every two days, CronTrigger won't work. To illustrate this, simply think of how many days are in a typical month (28-31). A cron expression like "0 0 5 2/2 * ?" would give us a trigger that would restart its count at the beginning of every month. This means that we would would get subsequent firings on July 30 and August 2, which is an interval of three days, not two.
+At first glance, you may be tempted to use a CronTrigger. However, if this is truly to be every two days, CronTrigger won't work. To illustrate this, simply think of how many days are in a typical month (28-31). A cron expression like "0 0 5 2/2 * ?" would give us a trigger that would restart its count at the beginning of every month. This means that we would get subsequent firings on July 30 and August 2, which is an interval of three days, not two.
```
Comment (jsoref) [inline docs/cookbook/BiDailyTrigger.md:14]: Duplicated words aren't always typos, but they often are, and I think all instances I saw in this repository were. Spell checkers have been performing this check for decades even though it isn't actually an instance of a word not in a dictionary.

File: docs/cookbook/DailyTrigger.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: Cookbook
 visible_title: "Quartz Cookbook"
 active_sub_menu_id: site_mnu_docs_cookbook
 ---
-<div class="secNavPanel"><a href=".">Contents</a> | <a href="NintyMinTrigger.html">&lsaquo;&nbsp;Prev</a> | <a href="BiDailyTrigger.html">Next&nbsp;&rsaquo;</a></div>
+<div class="secNavPanel"><a href=".">Contents</a> | <a href="NinetyMinTrigger.html">&lsaquo;&nbsp;Prev</a> | <a href="BiDailyTrigger.html">Next&nbsp;&rsaquo;</a></div>
```
Comment (jsoref) [inline docs/cookbook/DailyTrigger.md:6]: this file is renamed below

File: docs/cookbook/NinetyMinTrigger.md
Comment (jsoref) [inline docs/cookbook/NinetyMinTrigger.md:1]: This is the above mentioned rename

File: quartz/src/main/java/org/quartz/core/QuartzSchedulerMBeanImpl.java [hunk 4]
```diff
@@ -938,9 +938,9 @@ public MBeanNotificationInfo[] getNotificationInfo() {
      * @see javax.management.NotificationBroadcaster#addNotificationListener(javax.management.NotificationListener,
      *      javax.management.NotificationFilter, java.lang.Object)
      */
-    public void addNotificationListener(NotificationListener notif,
+    public void addNotificationListener(NotificationListener listener,
```
Comment (jsoref) [inline quartz/src/main/java/org/quartz/core/QuartzSchedulerMBeanImpl.java:941]: Sometimes `notif` was used for `notification` and sometimes for `listener`. By not abbreviating, things are clearer...

File: quartz/src/main/java/org/quartz/impl/JobDetailImpl.java [hunk 5]
```diff
@@ -386,7 +386,7 @@ public boolean isPersistJobDataAfterExecution() {
     /**
      * @return whether the associated Job class carries the {@link DisallowConcurrentExecution} annotation.
      */
-    public boolean isConcurrentExectionDisallowed() {
+    public boolean isConcurrentExecutionDisallowed() {
```
Comment (jsoref) [inline quartz/src/main/java/org/quartz/impl/JobDetailImpl.java:389]: api break

File: quartz/src/main/java/org/quartz/impl/jdbcjobstore/CacheDelegate.java [hunk 6]
```diff
@@ -29,7 +29,7 @@
 
 /**
  * <p>
- * This is a driver delegate for Intersystems Caché database.
+ * This is a driver delegate for InterSystems Caché database.
```
Comment (jsoref) [inline quartz/src/main/java/org/quartz/impl/jdbcjobstore/CacheDelegate.java:32]: brand

File: quartz/src/main/java/org/quartz/impl/jdbcjobstore/JTANonClusteredSemaphore.java [hunk 7]
```diff
@@ -70,15 +70,15 @@ public class JTANonClusteredSemaphore implements Semaphore {
      * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      */
 
-    public static final String DEFAULT_TRANSACTION_MANANGER_LOCATION = "java:TransactionManager";
+    public static final String DEFAULT_TRANSACTION_MANAGER_LOCATION = "java:TransactionManager";
```
Comment (jsoref) [inline quartz/src/main/java/org/quartz/impl/jdbcjobstore/JTANonClusteredSemaphore.java:73]: api break?

File: quartz/src/main/java/org/quartz/listeners/BroadcastJobListener.java [hunk 8]
```diff
@@ -37,7 +37,7 @@
  * @see #removeListener(org.quartz.JobListener)
  * @see #removeListener(String)
  *
- * @author James House (jhouse AT revolition DOT net)
+ * @author James House (jhouse AT revolution DOT net)
```
Comment (jsoref) [inline quartz/src/main/java/org/quartz/listeners/BroadcastJobListener.java:40]: Oops, this is an email address. I have filters for email addresses, but they don't cover AT...DOT things...

File: quartz/src/main/java/org/quartz/utils/CircularLossyQueue.java [hunk 9]
```diff
@@ -105,7 +105,7 @@ public T peek() {
      * 
      * @return true if the queue is empty, false otherwise
      */
-    public boolean isEmtpy() {
+    public boolean isEmpty() {
```
Comment (jsoref) [inline quartz/src/main/java/org/quartz/utils/CircularLossyQueue.java:108]: api break

File: quartz/src/test/java/org/quartz/TriggerBuilderTest.java [hunk 10]
```diff
@@ -83,14 +83,14 @@ public void testTriggerBuilder() throws Exception {
...
-        assertTrue("Unexpected trigger priortiy: " + trigger, trigger.getPriority() == 2);
+        assertTrue("Unexpected trigger priority: " + trigger, trigger.getPriority() == 2);
         assertTrue("Unexpected start-time: " + trigger.getStartTime(), trigger.getStartTime().equals(stime));
         assertTrue("Unexpected end-time: " + trigger.getEndTime(), trigger.getEndTime() != null);
         
     }
     
     /** QTZ-157 */
-    public void testTriggerBuilderWithEndTimePriorCurrrentTime() throws Exception {
+    public void testTriggerBuilderWithEndTimePriorCurrentTime() throws Exception {
```
Comment (jsoref) [inline quartz/src/test/java/org/quartz/TriggerBuilderTest.java:93]: api break

File: quartz/src/test/java/org/quartz/core/ListenerManagerTest.java [hunk 11]
```diff
@@ -177,15 +177,15 @@ public void testManagementOfSchedulerListeners() throws Exception {
...
         for(int i=0; i < numListenersToTestOrderOf; i++) {
-        	lstners[i] = new TestSchedulerListener();
-        	manager.addSchedulerListener(lstners[i]);
+        	listeners[i] = new TestSchedulerListener();
+        	manager.addSchedulerListener(listeners[i]);
         }
         List<SchedulerListener> mls = manager.getSchedulerListeners();
         int i = 0;
-        for(SchedulerListener lsnr: mls) {
-        	assertSame("Unexpected order of listeners", lstners[i], lsnr);
```
Comment (jsoref) [inline quartz/src/test/java/org/quartz/core/ListenerManagerTest.java:188]: I don't see value in dropping letters here

File: quartz/src/test/java/org/quartz/integrations/tests/QuartzDatabasePauseAndResumeTest.java [hunk 12]
```diff
@@ -32,7 +32,7 @@
 /**
  * Created by zemian on 10/25/16.
  */
-public class QuartzDatabsePauseAndResumeTest extends QuartzDatabaseTestSupport {
+public class QuartzDatabasePauseAndResumeTest extends QuartzDatabaseTestSupport {
```
Comment (jsoref) [inline quartz/src/test/java/org/quartz/integrations/tests/QuartzDatabasePauseAndResumeTest.java:35]: api break

File: quartz/src/main/java/org/quartz/simpl/RAMJobStore.java [hunk 13]
```diff
@@ -779,10 +779,10 @@ public boolean removeCalendar(String calName)
...
 
         synchronized (lock) {
-            for (TriggerWrapper trigger : triggersByKey.values()) {
-                OperableTrigger trigg = trigger.trigger;
-                if (trigg.getCalendarName() != null
-                        && trigg.getCalendarName().equals(calName)) {
+            for (TriggerWrapper wrapper : triggersByKey.values()) {
+                OperableTrigger trigger = wrapper.trigger;
+                if (trigger.getCalendarName() != null
+                        && trigger.getCalendarName().equals(calName)) {
```
Comment (jsoref) [inline quartz/src/main/java/org/quartz/simpl/RAMJobStore.java:785]: The outer thing is in fact a `wrapper` and not a `trigger` using `trigg` was misleading.

**Comment by jsoref** [comment]
I'd be shocked if it weren't.

What's blocking it is a lack of 👀 .

File: quartz/src/test/java/org/quartz/integrations/tests/QTZ336_MissSchedulingChangeSignalTest.java [hunk 14]
```diff
@@ -117,8 +117,8 @@ public void simpleScheduleAlwaysFiredUnder20s() throws Exception {
      * condition the job must be real quick and not allowing concurrent executions.
      */
     @DisallowConcurrentExecution
-    public static class CollectDuractionBetweenFireTimesJob implements Job {
-        private static final Logger log = LoggerFactory.getLogger(CollectDuractionBetweenFireTimesJob.class);
+    public static class CollectDurationBetweenFireTimesJob implements Job {
```
Comment (Noitcereon) [inline quartz/src/test/java/org/quartz/integrations/tests/QTZ336_MissSchedulingChangeSignalTest.java:120]: api break?

File: quartz/src/main/java/org/quartz/core/jmx/QuartzSchedulerMBean.java [hunk 15]
```diff
@@ -183,7 +183,7 @@ boolean deleteJob(String jobName, String jobGroupName)
      * Maps are limited to containing values from the same set of types, less Map itself.
      * 
      * @param abstractJobInfo map of attributes defining job
-     * @param replace whether or not to replace a pre-existing job with the same key
+     * @param replace whether or not to replace a preexisting job with the same key
```
Comment (Noitcereon) [inline quartz/src/main/java/org/quartz/core/jmx/QuartzSchedulerMBean.java:186]: I think both pre-existing and preexisting are correct, but I guess not a major deal. (https://dictionary.cambridge.org/dictionary/english/pre-existing)

File: quartz/src/main/java/org/quartz/core/QuartzScheduler.java [hunk 16]
```diff
@@ -1993,7 +1993,7 @@ public void notifySchedulerListenersError(String msg, SchedulerException se) {
         }
     }
 
-    public void notifySchedulerListenersSchduled(Trigger trigger) {
+    public void notifySchedulerListenersScheduled(Trigger trigger) {
```
Comment (Noitcereon) [inline quartz/src/main/java/org/quartz/core/QuartzScheduler.java:1996]: Api break?

**Review by Noitcereon — COMMENTED** [review]
I think it looks good overall, but instead of just writing "A number of api breaks" in the PR, perhaps list all of them explicitly to ease any transition from the incorrectly spelled versions.

**Comment by jsoref** [comment]
@Noitcereon: I've updated the PR description to list the things that I suspect are API breaks.

**Review by Noitcereon — COMMENTED** [review]
I guess the next step (I'm assuming here, as I am new to the project) is to contact one of the maintainers to look/approve it.

https://github.com/quartz-scheduler/contributing/blob/main/quartz-scheduler_publishers.txt

**Comment by jsoref** [comment]
@jhouserizer ?

**Review by t-beckmann — COMMENTED** [review]
Did not find any errors. There are API breaking corrections of typos that are very easy and obvious to fix. The author spent effort and did a good job. +1

**Comment by jhouserizer** [comment]
Unfortunately, due to the change of ownership of this project, the terms of contribution have been updated, and this PR will need to be re-created, agreeing to the new terms.

**Pull request closed without merging**